SameLiveDataException.java

```diff
@@ -0,0 +1,9 @@
+package com.tsuchiya.live;
+
+ // 既存のレコードと同じライブ情報を更新しようとした場合にスローするための例外処理クラス。
+public class SameLiveDataException extends RuntimeException {

```

Comment:
ニュアンスとしては、以下のような例外の名前の方がみますね
```suggestion
public class DuplicateLiveDataException extends RuntimeException {

```

Reply:
SameLiveDataException → DuplicateLiveDataExceptionに変更しました。
変更箇所はこちらです。
[GlobalExceptionHandler.java](https://github.com/kttsu/learning_task_10/pull/3/commits/04ebc66d53588678bd5d391cd4bf21dbee2ad176#diff-e709bf244c1cbd15ca1b9146248a623866037747ec131cd07af11965f3b2660c)